src/components/Autocomplete/index.jsx

```diff
@@ -332,7 +317,6 @@ export default class Autocomplete extends React.Component {
       return (
```

Comment:
lines 229 and 304: add theme()?

Reply:
For 229 yes! I will add this, but for 304 I'm not sure how to do this right now. I wanted to just leave it and see if anyone has a need to theme the highlight and if so I can think about a solution then.

Reply:
sounds good